sherpa/astro/ui/utils.py

```diff
@@ -12364,7 +12367,7 @@ def calc_kcorr(self, z, obslo, obshi, restlo=None, resthi=None,
     # Session Text Save Function
     ###########################################################################
 
-    def save_all(self, outfile=None, clobber=False):
+    def save_all(self, outfile=None, clobber=False, outfh=None):
```

Comment:
tInstead of having to introduce a new argument and conditional checks on arguments combinations (e.g. if filename is None then use the file handler, if it is not None, etc.), couldn't we just overload the outfile argument and make it hold either None (stdout), or a string (filename), or an object (file handler)?